### PR TITLE
feat: osano banner in storybook

### PR DESCRIPTION
**Pull request by burakukula**
Osano banner in storybook. 
🍩  pretending to be 🍪 . WDYT?

Screenshot 2021-08-10 at 14 28 56
Screenshot 2021-08-10 at 14 29 07
Screenshot 2021-08-10 at 14 29 20

### CODE REVIEW

File: scripts/.storybook/manager.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { addons } from '@storybook/addons';
 import contentfulTheme from './contentful-theme';
+import '!style-loader!css-loader!./osano.css';
```
Comment (burakukula) [inline scripts/.storybook/manager.js:3]: I found this solution in [this disussion](https://github.com/storybookjs/storybook/issues/6364). This is the only way we were able to make it work 🤷‍♀️

File: scripts/.storybook/manager-head.html [hunk 2]
```diff
@@ -1 +1,2 @@
 <link rel="icon" type="image/png" href="/favicon.png" sizes="32x32" />
+<script src="https://cmp.osano.com/16BcqiRsJId123ATa/6be6855b-d6ed-4ae6-83c6-d1757f311bd6/osano.js"></script>
```
Comment (burakukula) [inline scripts/.storybook/manager-head.html:2]: Those keys are available in the source of contentful website, same in Forma 36 website. I guess it's ok for us to have it in the repository?

File: scripts/.storybook/osano.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
+@import '~@contentful/forma-36-tokens/dist/css/index.css';
```
Comment (suevalov) [inline scripts/.storybook/osano.css:1]: can't we use `@import '~@contentful/experience-consent-manager/dist/style.css'`?

File: scripts/.storybook/osano.css [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+/* Widget */
+.osano-cm-widget {
+  cursor: pointer;
+  border-radius: 20%;
+  transition: none;
+  box-sizing: border-box;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-color: var(--color-white);
+  background-image: url('./public/DonutVelCookie.png');
```
Comment (bgutsol) [inline scripts/.storybook/osano.css:19]: I think we can reduce the size of this cookie image by half.

**Review by bgutsol — APPROVED** [review]
All good 👍
Only one small comment

**Review by massao — APPROVED** [review]

File: scripts/.storybook/osano.css (hunk 3 above)
Comment (burakukula) [reply scripts/.storybook/osano.css:1]: @suevalov `@contentful/experience-consent-manager` is is like a separate package? I can not find it 😬

Comment (suevalov) [reply scripts/.storybook/osano.css:1]: yep, https://github.com/contentful/creator-app-mvp/blob/master/package.json#L54

**Review by thebesson — APPROVED** [review]

File: scripts/.storybook/osano.css (hunk 3 above)
Comment (burakukula) [reply scripts/.storybook/osano.css]: @suevalov it worked, just needed to overwrite some stuff that was overwritten in the  `experience-consent-manager` 🤪

**Review by suevalov — APPROVED** [review]

**Pull request merged**